doc/ref_arch/openstack/chapters/chapter04.md

```diff
@@ -25,21 +25,17 @@
 
 <a name="4.1"></a>
 ## 4.1 Introduction.
-**Will update this section with a summary from the sub-chapters**
 
-<!--
-Chapter 3 presented the high level architecture and core OpenStack services for creating an IaaS cloud. This chapter discusses the second level of details (as defined by L3) including deployment topology, distribution of the core OpenStack services among Controller and Compute nodes.
+Chapter 3 introduced the components of an OpenStack-based IaaS
+-	Consumable Infrastructure Resources and Services
+-	NFVI Management Software (VIM: OpenStack) core Services and architectural constructs needed to consume and manage the Consumable resources
```

Comment:
**nit:** There are some capitalised words in the middle of the sentence (which are not abbreviation or a component name/noun)
Services -> **s**ervices
Consumable -> **c**onsumable

Reply:
@msidana Corrected -- Thanks